packages/workspace/src/core/hasher/git-hasher.ts

```diff
@@ -132,10 +132,9 @@ function checkForDeletedFiles(
       statSync(join(path, f)).isFile();
       filesToHash.push(f);
     } catch {
-      console.warn(
-        `Warning: Fell back to using 'fs' to identify ${f} as deleted. Please open an issue at https://github.com/nrwl/nx so we can investigate.`
+      throw new Error(
+        `Error: Fell back to using 'fs' to identify ${f} as deleted. Please open an issue at https://github.com/nrwl/nx so we can investigate.`
       );
```

Comment:
I see this warning many times with my daily usage of Nx, will it have no side effect to turn it into an error?

Reply:
@edbzn That’s interesting and good to know. Have you reported it as an issue as the warning suggests?

As you can see in the description of the PR we based this decision on having not heard from anyone for a long time that this was occurring.

Please provide as much information as possible so that we can help troubleshoot

Reply:
@JamesHenry I will report it if I face it again. From what I remember it was when deleting/moving files in my workspace but I will fill an issue when I'm able to reproduce it.